### PR TITLE
Show wiki to target, ie. Board---FRSKYF4

### PR DESCRIPTION
**Pull request by HThuren**
To support the description of Boards at Wiki, you got with this PR an annoncement of [wiki] when download a FC target.
You must in unified target board configuration make a line with ex: `#wiki https://github.com/betaflight/betaflight/wiki/Board---KAKUTEH7`, this make the Wiki to refer to `https://github.com/betaflight/betaflight/wiki/Board---KAKUTEH7`

Before change
image

After change and #wiki defined in unified target board configuration  or if not #wiki are defined, a general link to wiki
image

### CODE REVIEW

**Comment by haslinghuis** [comment]
Any screenshots would be nice.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by HThuren** [comment]
> Any screenshots would be nice (also you didn't update your fork before submit ;)

Screenshots provided.
Sorry for the update, I did believe that was an andvantage to easy merge, but seems not

**Comment by haslinghuis** [comment]
@blckmn with this we can document each board. Only have to choose between `Board---NAME` or `Board - NAME`

We might remove F3 from wiki and should rename some pages to reflect standard name format.

**Comment by HThuren** [comment]
Also notice some board descriptions are located at https://github.com/betaflight/betaflight/blob/master/docs/boards with "Board-" + target as name format (1 -, not 3). But they seems to be old stuff (?)

**Comment by blckmn** [comment]
Another option here is to add wiki details to the board config files

#wiki [url]
#url  [url] 

That way I can pick it up in parsing and return it as a property in 'summary'

Can provide both a manufacturers product url and out wiki

**Comment by blckmn** [comment]
image

Also can I suggest instead of an entirely new line, we simply place it after the Target name?

**Comment by HThuren** [comment]
I think this as two good suggestions. I can change this PR for presentation at first line, with mouse_over message as well.
I only need the property name in summary to a start.

**Comment by blckmn** [comment]
Too easy. summary.wiki though it's only currently populated for FRSKYF4 you're testing with.

**Comment by HThuren** [comment]
@blckmn, by the way, how do I get your change

File: src/js/tabs/firmware_flasher.js
Comment (haslinghuis) [inline src/js/tabs/firmware_flasher.js:72]: If we are going to set this up anew then we could start with just single `-` ??? @blckmn

Comment (blckmn) [reply src/js/tabs/firmware_flasher.js:72]: No just use the property `summary.wiki` if it is defined then display the wiki link, else hide it.

Comment (blckmn) [reply src/js/tabs/firmware_flasher.js:72]: so something like this: 
```
if (summary.wiki) {
     targetWiki.html('[Wiki]');
     targetWiki.attr("href", summary.wiki);
} else {
    targetWiki.html('');
} 
```

Comment (HThuren) [reply src/js/tabs/firmware_flasher.js:72]: No issue to change number of dash, but maybe just ex. [Wiki: Board FRSKYF4]", this to promote the benefits of documenation at the Wiki. This may also give the code snip:
     targetWiki.html('[Wiki]');
     if (summary.wiki) {
          targetWiki.attr("href", summary.wiki);
     } else {
         targetWiki.html("href", 'https://github.com/betaflight/betaflight/wiki');  // can be an entry in message.jason
     }

File: locales/en/messages.json
Comment (blckmn) [inline locales/en/messages.json:3055]: Is this the right place to store this? Are we assuming there will be different wikis dependent on language?

File: src/js/tabs/firmware_flasher.js
Comment (blckmn) [inline src/js/tabs/firmware_flasher.js:75]: See above comment on whether the URL should be in the language messages file.

File: src/js/tabs/firmware_flasher.js
Comment (blckmn) [inline src/js/tabs/firmware_flasher.js:72]: I think we can probably just say "Wiki" - and not put the target in, as the target is already shown to the left

File: locales/en/messages.json
Comment (HThuren) [reply locales/en/messages.json:3055]: Well, just not to have an external defines in code but placed outside, but you are right this are not a language issue. No problem to change.

File: src/js/tabs/firmware_flasher.js
Comment (HThuren) [reply src/js/tabs/firmware_flasher.js]: Well, now changed as desired :-)

**Review by haslinghuis — APPROVED** [review]

**Comment by HThuren** [comment]
Hi @blckmn, are the solution all right now ?

**Review by blckmn — APPROVED** [review]

**Comment by HThuren** [comment]
Thank you

**Pull request merged**
